NEARFlags.json

```diff
@@ -0,0 +1,31 @@
+{
+  "EXAMPLE_FLAG": {
```

Comment:
given that each flag description is pretty sizeable, seems like it makes sense to do one feature per file to allow easier merging and editing

Reply:
Related to your point around git blame -- it's a lot easier to track additions and removals to a file then the addition and removal of entire files inside of a directory, when doing forensics it's nice to be able to continually step back through the feature json file without needing to use a tool that is good at this (Webstorm does it very well, but I'm keenly aware that many people don't use it :))

With the flags organized alphabetically there will almost never be merge conflicts in my experience using this type of pattern previously on teams with 20+ devs committing to the same repo.  What is your concern around ease of editing here?

What do you think about creating 1 file _per environment_ to split things up a bit without creating a new file for every flag?